tmt-app/src/Components/ResultCards.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Fragment } from "react";
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Typography,
+  Container,
+} from "@mui/material";
+import { ReactComponent as EmotionalIntelligence } from "../Images/icons/emotional_intelligence.svg";
```

Comment:
Can we add this to the sprite sheet since we ended up adding the work to this PR?